daemon/cluster/executor/container/adapter.go

```diff
@@ -530,3 +530,14 @@ func isUnknownContainer(err error) bool {
 func isStoppedContainer(err error) bool {
 	return strings.Contains(err.Error(), "is already stopped")
 }
+
+func isErrContainerUnhealthy(err error) bool {
+	if err != nil {
+		if e, ok := err.(*exitError); ok {
```

Comment:
Should this use `errors.As()`? Something like;

```suggestion
	var eErr exitError
	if errors.As(err, & eErr) {
```

Perhaps use an early return for the `err == nil` case 🤔

Reply:
Yes, will do.

Reply:
Done.